packages/fxa-settings/src/styles/tailwind.css

```diff
@@ -24,3 +24,15 @@ body {
     @apply text-blue-800;
   }
 }
+
+.smooth-transition {
```

Comment:
Because this class is only used in a single file with elements that already have a wide variety of classes applied to them, is this helper class really necessary?

Reply:
I expect we're going to use this in every place where hovering changes some style, which is going to be quite a lot. These same styles are being applied to the ctas (I didn't use `@apply` there since I'd need to extract this into its own file and import it there, plus it doesn't seem best practice to `@apply` your own custom classes). This actually should also be applied to `link-blue`.

Do you think this makes a case for `@apply smooth-transition`? I just don't like the idea of, if we decide the transition should be longer or shorter, finding and replacing every instance of `transition-150` especially if we could have that set for some other animation, when we could do it in one place. 😩

Reply:
If this is really going to be used in multiple areas and not just the footer then leave it in. I'm just trying my best to keep this custom CSS as lean as possible. Yes, _if_ we change a transition or other property it would be more effort than necessary, and I will say I'm less concerned about this class because it's covering a specific area (transitions, and not a broad collection of properties), but I still hesitate to let a single class implement multiple properties when one of the main points of Tailwind is to be using the individual classes to style individual properties.

As an aside, but not too concerned (and more of a trolling), I don't know that `smooth` is necessary since we're likely not going to have a `bumpy` or `jagged` transition. It could simply be `standard-transition`.

Reply:
I do like `standard-transition` better.

I feel like having some custom component classes is to be expected at the beginning of a Tailwind project, there's even a [section of their docs](https://tailwindcss.com/docs/extracting-components/#extracting-css-components-with-apply) that recommends doing this. I understand wanting to keep things lean, but I also don't want to have a design request of updating something, and then it not be reflected across the board because we have duplication that we didn't catch. I understand your concern, but I don't think what we have so far is over the top extraction. 

And oh, I can certainly take the transition styles off of the ctas and make sure the ctas use `cta-neutral standard-transition` (or whatever cta class applies). That'll help a bit.